src/IO/S3/copyS3File.cpp

```diff
@@ -15,6 +15,10 @@
 
 namespace ProfileEvents
 {
+    extern const Event WriteBufferFromS3Bytes;
+    extern const Event WriteBufferFromS3Microseconds;
+    extern const Event WriteBufferFromS3RequestsErrors;
```

Comment:
This metrics should be added to a single part upload as well

Reply:
Done. Thanks